charts/argo-cd/README.md

```diff
@@ -64,7 +64,170 @@ applicationSet:
   replicas: 2
 ```
 
-### Synchronizing Changes from Original Repository
+## Ingress configuration
+
+Please refer to [Operator Manual](https://argo-cd.readthedocs.io/en/stable/operator-manual/ingress/#ingress-configurationh) for details as samples below
+corespond to respetive sections.
+
+### SSL-Passthrough
+
+The `tls: true` option will expect that `argocd-server-tls` secret exists as Argo CD server loads TLS certificates from this place.
```

Comment:
should be "will expect that the `argocd`..."

Reply:
Fixed